locations/spiders/smashburger.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import re
+
+import scrapy
+import json
+
+from locations.items import GeojsonPointItem
+
+class SmashburgerSpider(scrapy.Spider):
+    download_delay = .2
+    name = "smashburger"
+    item_attributes = {'brand': "Smashburger"}
+    allowed_domains = ["smashburger.com"]
+    start_urls = (
+        'https://api.smashburger.com/mobilem8-web-service/rest/storeinfo/distance?_=1649446017671&attributes=&disposition=PICKUP&latitude=39.7392358&longitude=-104.990251&maxResults=50&radius=100&radiusUnit=mi&statuses=ACTIVE,TEMP-INACTIVE&tenant=sb-us',
+    )
+
+    def parse(self, response):
```

Comment:
Look at other spider examples that use the `def start_requests(self)` function to do this.